prebid-server/endpoints/openrtb2/video.md

```diff
@@ -0,0 +1,355 @@
+6. The SSAI server parses the returned key-values, appending them as a query string to the ad server request URL and submits the request. 
+7. The ad server returns the optimized pod. 
+8. The SSAI server requests the creatives from prebid cache.  
+9. The SSAI server requests the content from the content host and stitches the creatives and content together. 
+10. The stitched stream is returned to the application. 
+
+<br>
+<img src="/assets/images/flowcharts/pb-lfv-serverside.png">
+<br>
+
+ **Parameters**<a name="parameters"></a>
+
+  {: .table .table-bordered .table-striped }
+| Param | Scope | Type | Description |
+| --- | --- | --- | --- |
+| storedrequestid | Optional  (Required for Xandr PBS) | `String` |  The id used to look up the entire request from a stored configuration in the database. |
+| podconfig | Required | `Object` |  Container object for describing all the pod configurations. |
+| podconfig.durationrangesec | Required | `Integer[]` |  Range of ad durations allowed in the response. See [pod duration range](#pod-duration-range) for more details. |
+| podconfig.requireexactduration | Optional | `Boolean` |  Flag indicating exact ad duration requirement. Default is false. |
+| podconfig.pods | Required | `Object[]` |  Container object for describing the adPod(s) to be requested.  |
+| site | Required | `Object` |  Container object for the URL of the impression to be shown. Can be excluded if app is included.  |
+| site.page | Required | `String` | URL of the page where the impression will be shown.  |
+| app | Required | `Object` |  Container object for app information where the impression is to be shown. Can be excluded if site is included.  |
+| app.domain | Required | `String` |  The domain of the app. |
+| app.name | Optional | `String` |  The name of the app. |
+| app.bundle | Optional | `String` |  A platform-specific application identifier intended to be unique to the app and independent of the exchange. On Android, this should be a bundle or package name (e.g., com.foo.mygame). On iOS, it is typically a numeric ID. |
+| app.storeurl | Optional | `String` |  App store URL for an installed app. |
+| video | Required | `Object` |  Container object for video player data. |
+| video.w | Optional | `Integer` |  Width of the video player in device independent pixels. |
+| video.h | Optional | `Integer` |  Height of the video player in device independent pixels. |
+| video.mimes | Required | `String[]` |  Player supported mime types. Example: ["video/mp4"] |
+| video.protocols | Required | `Integer[]` |  Array of supported [video protocols](#video-protocols). |
+| includebrandcategory | Optional | `Object` |  Container Object for passing Category inputs. |
+| includebrandcategory.primaryadserver | Optional | `Integer` |  The ad server used by the publisher. Supported Values 1- Freewheel , 2- DFP. |
+| includebrandcategory.publisher | Optional | `Integer` |  Identifier for the Publisher. |
```

Comment:
The `includebrandcategory.publisher` field uses a string value currently in the code, so it doesn't need to be limited to just integers.

Reply:
Changed the type to string